### ISSUE TITLE
Template: Hamburger menu button does not have visible focus

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#91]
In mobile view, the hamburger menu image to open the menu does not receive a visible focus.

ensure focus items are visible AND items that do not need to be visible do not receive focus.

**Comment by syarwick23** [comment]
Issue fixed with new template
